client/galaxy/scripts/components/Workflow/Editor/Node.vue

```diff
@@ -1,9 +1,10 @@
 <template>
     <div>
-        <div class="toolFormTitle unselectable clearfix">
-            <i :class="iconClass" />
-            <span class="node-title">{{ title }}</span>
-            <span class="sr-only">&nbsp;Node</span>
+        <div class="node-header unselectable clearfix">
+            <div class="header float-left">
```

Comment:
Without additional styles to fix the display, this can cause tool titles to wrap where they previously would not.  Merging anyway since there's the more style-focused followup PR we can address this in.